target/linux/mvebu/image/cortex-a9.mk

```diff
@@ -183,4 +183,20 @@ define Device/cznic_turris-omnia
 endef
 TARGET_DEVICES += cznic_turris-omnia
 
+define Device/wd_mcex2u
+  $(Device/NAND-128K)
+  DEVICE_TITLE := WD My Cloud EX2 Ultra (BVBZ/Ranger Peak)
+  DEVICE_PACKAGES += wtmcu -uboot-envtools -dnsmasq -ppp -ppp-mod-pppoe \
+    block-mount e2fsprogs partx-utils mkf2fs f2fsck \
+    kmod-dm kmod-md-mod kmod-fs-vfat kmod-fs-ext4 kmod-fs-xfs \
+    kmod-usb-ledtrig-usbport kmod-usb2 kmod-usb3 kmod-usb-storage kmod-usb-xhci
+  DEVICE_DTS := armada-385-wd-mcex2u
+  KERNEL_SIZE := 5120k
+  IMAGES += image.cfs-factory.img
+  IMAGE/image.cfs-factory.img := append-ubi
+  IMAGES += uImage-factory.img
```

Comment:
I'd prefer a more consistent scheme for naming these images.

Reply:
They are named after the partitions on the device (original DTS) in order to help with installation. What can you propose for better names?

Reply:
```
0x000000000000-0x000000500000 : "U-Boot"
0x000000500000-0x000000a00000 : "uImage"
0x000000a00000-0x000000f00000 : "uRamdisk"
0x000000f00000-0x00000c800000 : "image.cfs"
0x00000c800000-0x00000d700000 : "rescue fw"
0x00000d700000-0x00000eb00000 : "config"
0x00000eb00000-0x00000f500000 : "reserve1"
0x00000f500000-0x00000ff00000 : "reserve2"
```

Reply:
Ah, now I see. I really don't like the dot for the first one, as it looks like you apply two different styles here.

I'd consider just replacing the dot by a hyphen, which will preserve your initial idea but won't look so strange IMO:
`IMAGE/image-cfs-factory.img`

Despite, I'd add the IMAGES in one line, followed by the definitions:
```
IMAGES += image-cfs-factory.img uImage-factory.img
IMAGE/image-cfs-factory.img := ...
IMAGE/uImage-factory.img := ...
```

I'd personally prefer .bin to .img, but I'm not sure what's common here on mvebu.